pkg/apis/storage/types.go

```diff
@@ -282,6 +282,8 @@ type CSIDriverSpec struct {
 	// Refer to the specific FSGroupPolicy values for additional details.
 	// This field is alpha-level, and is only honored by servers
 	// that enable the CSIVolumeFSGroupPolicy feature gate.
+	// Defaults to ReadWriteOnceWithFSType, which will use heuristics to
+	// determine if Kubernetes should modify ownership and permissions of the volume.
```

Comment:
I'm trying to read the definition of this and it's not really exactly clear what it does.  For instance, the name includes `Once`, but it's not clear from the description below what decides "once", and I think it needs to.

Reply:
I've updated the definition in this section to be more explicit and have it match the description of the type below. This will attempt to modify volume permissions if both a `fstype` is defined and the access modes contain `ReadWriteOnce`.

Reply:
@smarterclayton ,

Is it possible to take a look at the latest definition to determine if this is clearer? I believe this is ready for another review.